bundler/lib/bundler/inline.rb

```diff
@@ -41,6 +41,7 @@ def gemfile(install = false, options = {}, &gemfile)
     Bundler.instance_variable_set(:@bundle_path, Pathname.new(Gem.dir))
     old_gemfile = ENV["BUNDLE_GEMFILE"]
     Bundler::SharedHelpers.set_env "BUNDLE_GEMFILE", "Gemfile"
+    Bundler::SharedHelpers.major_deprecation 2, "The 'install' argument is deprecated." if install
```

Comment:
Also, we'll want to explain why, and recommend the alternative (don't pass anything).